docs/clients_tfs.md

```diff
@@ -129,24 +154,45 @@ When creating a Python-based client application, there are two packages on PyPi
             ("outputs", outputs_metadata)
         ])
 
+.. tab:: curl
+
+    .. code-block:: sh  
+
+        curl http://localhost:9000/v1/models/my_model/metadata
+
+
 @endsphinxdirective
 
 ### Request Prediction on a Binary Input
 
 @sphinxdirective
 
-.. tab:: ovmsclient
+.. tab:: ovmsclient [GRPC]
 
     .. code-block:: python
 
         from ovmsclient import make_grpc_client
 
-        client = make_grpc_client("10.20.30.40:9000")
+        client = make_grpc_client("localhost:9000")
         with open("img.jpeg", "rb") as f:
             data = f.read()
         inputs = {"input_name": data}    
         results = client.predict(inputs=inputs, model_name="my_model")
 
+.. tab:: ovmsclient [REST]
+
+    .. code-block:: python
+
+        from ovmsclient import make_http_client
+
+        client = make_http_client("localhost:9000")
+        model_metadata = client.get_model_metadata(model_name="my_model", model_version="", 10.0)
+
+        with open("img.jpeg", "rb") as f:
+            data = f.read()
+        inputs = {"input_name": data}    
+        results = client.predict(inputs=inputs, model_name="my_model", model_version="", 10.0)
```

Comment:
Same as for status. 
port could be different and additional parameters are not relevant here